.eslintrc.js

```diff
@@ -231,5 +231,7 @@ module.exports = {
     '/packages/graphql-mapping-template/lib',
     '/packages/graphql-*-transformer/lib',
     '/packages/graphql-transformer-*/lib',
+    '/packages/amplify-headless-interface/lib',
+    '/packages/amplify-util-headless-input/lib'
```

Comment:
It definitely shouldn't be part of the individual categories because then it would have to be duplicated for each category. I'm of the opinion that we should be trying to break apart the massive amplify context object into smaller libraries rather than putting more into it which is why I made it a separate package. For customers writing headless scripts, they may also want to import the schema validation logic without importing the whole CLI

Reply:
Discussed offline. We can leave the validation logic in a the util-headless package for now.